x-pack/legacy/plugins/upgrade_assistant/server/np_ready/lib/apm/index.ts

```diff
@@ -46,7 +46,9 @@ export const isLegacyApmIndex = (
   apmIndexPatterns: string[] = [],
   mappings: FlatSettings['mappings']
 ) => {
-  const clientVersion = new SemVer(get(mappings, '_meta.version', '0.0.0'));
+  const defaultValue = '0.0.0';
+  const version = get(mappings, '_meta.version', defaultValue);
+  const clientVersion = new SemVer(valid(version) ? version : defaultValue);
```

Comment:
On the 7.x branch, these APM indices are correctly being identified and showing up as a different issue in the UI.

While I haven't confirmed it, it appears that this change may have broken this functionality. 

### Correct Behavior (7.x)

<img width="1238" alt="Screen Shot 2019-12-02 at 4 36 18 PM" src="https://user-images.githubusercontent.com/1813008/70001201-5194d080-1522-11ea-9dd0-3f50f1f82fac.png">


### Incorrect (this branch)

<img width="1237" alt="Screen Shot 2019-12-02 at 4 35 12 PM" src="https://user-images.githubusercontent.com/1813008/70001196-4e014980-1522-11ea-8b62-29fa8d31195f.png">


### Test data

Use [this archive](https://github.com/elastic/kibana/files/3913596/data-7-0-apm.tar.gz) to initialize Elasticsearch with some test data:

```
yarn es snapshot --data-archive path/to/data-7-0-apm.tar.gz
```

Reply:
Great catch and thanks for the archive! Looks like my shim was missing `server.plugins.apm_oss` which was causing this.